hack/sync-api.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage() {
+    cat <<EOF
+ENV:
+    API_ROOT: clusterpedia api path, required if the pwd is not the api path.
+        eg. API_ROOT=/home/runner/work/clusterpedia/staging/src/github.com/clusterpedia-io/api
+    GH_TOKEN: github token for api repo auth.
+EOF
+}
+
+set +e; REPO_ROOT=$(git rev-parse --show-toplevel 2>/dev/null);set -e
+if [ -z $REPO_ROOT ]; then
+    if [ -z $API_ROOT ]; then
+        echo "the current directory is not in the clusterpedia api path, please set API_ROOT=<clusterpedia api path>"
+        usage
+    exit 1
+    fi
+    REPO_ROOT=$(pwd)
+else
+    API_ROOT="${REPO_ROOT}/staging/src/github.com/clusterpedia-io/api"
+fi
+
+if [ -z $GH_TOKEN ]; then
+    echo "the github token is not in the env, please check CLUSTERPEDIA_BOT_TOKEN"
+    usage
+    exit 1
+else
+    API_REPO="https://$GH_TOKEN@github.com/clusterpedia/api.git"
+fi
+
+raw=$(git branch -r --contains $REF)
```

Comment:
The `git <cmd>` must be in REPO, right? 
If we can't get REPO_ROOT, the `git <cmd>` will definitely fail.

Reply:
it is fixed